internal/output/diff.go

```diff
@@ -80,7 +80,7 @@ func ToDiff(out Root, opts Options) ([]byte, error) {
 	if len(noDiffProjects) > 0 {
 		s += "──────────────────────────────────\n"
 		s += fmt.Sprintf("\nThe following projects have no cost estimate changes: %s", strings.Join(noDiffProjects, ", "))
-		s += fmt.Sprintf("\nRun %s to see their breakdown.", ui.PrimaryString("infracost breakdown"))
+		s += fmt.Sprintf("\nRun %s to see their breakdown.", ui.PrimaryString("infracost breakdown --path /path/to/code"))
```

Comment:
Let's encourage using the equals character for flags :)
```suggestion
		s += fmt.Sprintf("\nRun %s to see their breakdown.", ui.PrimaryString("infracost breakdown --path=/path/to/code"))
```